code/modules/reagents/reagents/medical.dm

```diff
@@ -565,7 +565,7 @@
 	scannable = TRUE
 
 /datum/reagent/medicine/peridaxon_plus/on_mob_life(mob/living/L, metabolism)
-	L.adjustToxLoss(20*REM)
+	L.reagents.add_reagent(/datum/reagent/toxin,5)
```

Comment:
Again. Undocumented.

Reply:
Perdiox+ altered. Toxin amount from normal use removed, but amount added to OD damage. Instead, normal usage generates 5 toxin (the chem) per tick (total of 50). Take antitox with it and there isn't a downside. Forget antitox and you'll have a bad day.